src/commands/init.js

```diff
@@ -0,0 +1,23 @@
+const { flags } = require('@oclif/command');
+const AbstractAuthenticatedCommand = require('../abstract-authenticated-command');
+const logger = require('../services/logger');
+
+class InitCommand extends AbstractAuthenticatedCommand {
+  async runIfAuthenticated() {
+    // TO BE REMOVED. JUST CHECKING IF THE FLAG PARSING IS WORKING HERE :)
+    const parsed = this.parse(InitCommand);
+    logger.info('working here 🤙 flag:', parsed.flags);
+  }
+}
+
+InitCommand.description = 'Set up your development environment in your current folder.';
+
+InitCommand.flags = {
+  project: flags.string({
+    char: 'p',
+    description: 'The name of the project you want to init.',
+    default: null,
```

Comment:
[Question] I'm not sure about this `default: null`. Might need to check, but I guess if you don't put this, you would simply not get the option if not given instead of getting `project=null`

Reply:
Ok I remove it - let's see later on if we'll need it.